app/serializers/item_serializer.rb

```diff
@@ -0,0 +1,27 @@
+class ItemSerializer < ActiveModel::Serializer
+  attributes :id, :description
+
+  belongs_to :list
+
+  def description
+    object.description
+  end
+end
+
+class ListSerializer < ActiveModel::Serializer
```

Comment:
@naimaseales There will be unpredictable behaviour with the serializers as some serializers are defined and then later redefined. For example, `ListSerializer` is first defined in this file `item_serializer.rb`, and then redefined in `list_serializer.rb`.

The serializers can be simplified to the following and give you the same behaviour. There should be no need to define a serializer more than once in this app at the moment:

```rb
# File: user_serializer.rb
class UserSerializer < ActiveModel::Serializer
  attributes :id, :created_at, :email
  has_many :lists

  def created_at
     # Prefer to use the ISO8601 internationally understood date format for APIs.
    object.created_at.iso8601
  end
end
```
```rb
# File: list_serializer.rb
class ListSerializer < ActiveModel::Serializer
  attributes :id, :title
  belongs_to :user
  has_many :items
end
```

```rb
# File: item_serializer.rb
class ItemSerializer < ActiveModel::Serializer
  attributes :id, :description
  belongs_to :list
end
```